controllers/datamover.go

```diff
@@ -558,6 +559,59 @@ func (r *DPAReconciler) createResticSecretsPerBSL(dpa *oadpv1alpha1.DataProtecti
 	return nil, nil
 }
 
+func (r *DPAReconciler) createResticCustomCASecret(dpa *oadpv1alpha1.DataProtectionApplication, bsl velerov1.BackupStorageLocation) (*corev1.Secret, error) {
+	insecureSkipTLSVerify, skipPresent := bsl.Spec.Config["insecureSkipTLSVerify"]
+	if !skipPresent || insecureSkipTLSVerify != "false" {
+		return nil, nil
+	}
+	if bsl.Spec.ObjectStorage.CACert == nil {
```

Comment:
This check would maybe be better here: https://github.com/openshift/oadp-operator/blob/master/controllers/bsl.go#L17

Reply:
That's a better place to fail if there's a configuration problem, but I still need to create the secret based on the value of insecureSkipTLSVerify. I pulled this out into its own function so it can be called from both places, if that makes sense: https://github.com/openshift/oadp-operator/pull/918/commits/3c55c383d78ec04e0585eef6e1ece18a73b16a41